README.md

```diff
@@ -14,16 +24,8 @@
 - [x] Does not lack features compared to proprietary app
 - [x] Does not need an account (the only exception is [Self-hosted](#self-hosted))
 - [x] Has dark theme
-
-### App Name ❤️ = Someone's Personal Favourite
-
-[More detailed explanation here.](RULES.md)
-
-[Why does it exist?](WHY.md)
-
-## Please use `Home` button to go to the top.
-
-__AWESOME__ apps counter: __187__ 🎉
```

Comment:
Everything is fine here, however I prefer to put `__AWESOME__ apps counter: __187__ 🎉` on the top of the `### Contents:`. The rest is fine.